praw/models/reddit/modmail.py

```diff
@@ -0,0 +1,46 @@
+"""Provide models for new modmail."""
+from ...const import API_PATH, urlparse
+from ...exceptions import ClientException
+from .base import RedditBase
+
+
+class ModmailConversation(RedditBase):
+    """A class for modmail conversations."""
+
+    STR_FIELD = 'id'
+
+    @staticmethod
+    def id_from_url(url):
```

Comment:
Wondering if getting modmail by URL is actually needed or wanted. Only submissions do this currently.